net-libs/aqbanking/aqbanking-6.5.4.recipe

```diff
@@ -0,0 +1,122 @@
+SUMMARY="A library for online banking and financial applications"
+DESCRIPTION="AQBanking"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://www.aquamaniac.de/rdm/attachments/download/499/aqbanking-$portVersion.tar.gz"
+CHECKSUM_SHA256="0d16ceae76f0718e466638f4547a8b14927f1d8d98322079cd6481adde30ac99"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="44.5.4"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	aqbanking$secondaryArchSuffix = $portVersion
+	lib:libaqbanking$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libgwenhywfar$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libxmlsec1$secondaryArchSuffix
+	lib:libxslt$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	aqbanking${secondaryArchSuffix}_devel = $portVersion
+	cmd:aqbanking_config$commandSuffix
+	devel:libaqbanking$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	aqbanking$secondaryArchSuffix == $portVersion base
+	devel:libgmp$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxmlsec1$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	devel:libgwenhywfar$secondaryArchSuffix
+	"
+
+SUMMARY_tools="Command line utilities for aqbanking"
+PROVIDES_tools="
+	aqbanking${secondaryArchSuffix}_tools = $portVersion
+	cmd:aqbanking_cli$commandSuffix
+	cmd:aqebics_tool$commandSuffix
+	cmd:aqhbci_tool4$commandSuffix
+	cmd:aqofxconnect_tool$commandSuffix
+	cmd:aqpaypal_tool$commandSuffix
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	aqbanking$secondaryArchSuffix == $portVersion base
+	lib:libgcrypt$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libgwenhywfar$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libxmlsec1$secondaryArchSuffix
+	lib:libxslt$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:xmlmerge$commandSuffix
+	devel:libgmp$secondaryArchSuffix
+	devel:libgwenhywfar$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxmlsec1$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize
```

Comment:
```suggestion
	cmd:libtoolize$secondaryArchSuffix
```

Reply:
Probably the reason it failed linking on 32bit (not finding -ltdl) :)